src/schema/v2/identity_verification.ts

```diff
@@ -0,0 +1,229 @@
+  result: string
+  extracted_id_fail_reason: string
+  extracted_similarity_fail_reason: string
+}
+
+const dateFieldForVerificationExpiresAt: GraphQLFieldConfig<
+  any,
+  ResolverContext
+> = {
+  ...dateField,
+  resolve: (
+    { invitation_expires_at: rawDate },
+    { format, timezone },
+    { defaultTimezone }
+  ) => {
+    const timezoneString = timezone || defaultTimezone
+    return formatDate(rawDate, format, timezoneString)
+  },
+}
+
+export const IdentityVerificationOverrideType = new GraphQLObjectType<
+  IdentityVerificationOverrideGravityResponse,
+  ResolverContext
+>({
+  name: "IdentityVerificationOverride",
+  fields: {
+    ...InternalIDFields,
+    newState: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "Overridden state",
+      resolve: ({ new_state }) => new_state,
+    },
+    oldState: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "Un-overridden state",
+      resolve: ({ old_state }) => old_state,
+    },
+    reason: { type: GraphQLString },
+    userID: {
+      description: "User ID of the override's creator",
+      type: new GraphQLNonNull(GraphQLString),
+      resolve: ({ user_id }) => user_id,
+    },
+    createdAt: {
+      type: GraphQLString,
+      resolve: ({ created_at }) => created_at,
+    },
+  },
+})
+
+export const IdentityVerificationScanReferenceType = new GraphQLObjectType<
+  IdentityVerificationScanReferenceGravityResponse,
+  ResolverContext
+>({
+  name: "IdentityVerificationScanReference",
+  fields: {
+    ...InternalIDFields,
+    jumioID: { type: GraphQLString },
+    extractedFirstName: {
+      type: GraphQLString,
+      resolve: ({ extracted_first_name }) => extracted_first_name,
+    },
+    extractedLastName: {
+      type: GraphQLString,
+      resolve: ({ extracted_last_name }) => extracted_last_name,
+    },
+    finishedAt: {
+      type: GraphQLString,
+      resolve: ({ finished_at }) => finished_at,
+    },
+    result: { type: GraphQLString },
+    extractedIdFailReason: {
+      type: GraphQLString,
+      resolve: ({ extracted_id_fail_reason }) => extracted_id_fail_reason,
+    },
+    extractedSimilarityFailReason: {
+      type: GraphQLString,
+      resolve: ({ extracted_similarity_fail_reason }) =>
+        extracted_similarity_fail_reason,
+    },
+  },
+})
+
+export const IdentityVerificationType = new GraphQLObjectType<
+  IdentityVerificationGravityResponse,
+  ResolverContext
+>({
+  name: "IdentityVerification",
+  fields: {
+    ...InternalIDFields,
+    state: {
+      type: new GraphQLNonNull(GraphQLString),
+      description: "Where the identity verification is in its lifecycle",
+    },
+    userID: {
+      description: "User ID of the identity verification's owner",
+      type: new GraphQLNonNull(GraphQLString),
+      resolve: ({ user_id }) => user_id,
+    },
+    invitationExpiresAt: dateFieldForVerificationExpiresAt,
+    overrides: {
+      type: new GraphQLList(IdentityVerificationOverrideType),
```

Comment:
Regarding lists, yup that was my intent. The APIs don't even offer pagination so I thought this would be appropriate.

Regarding `GraphQLNonNull`, I wasn't sure. Only admins are authorized to access the backing endpoints, so if you are a non-admin querying for a single verification of your own, you could attempt to query for these, but they'd come back as `null` along with an error. Given that, how should this type work?